src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -33,54 +23,15 @@ $popover-max-width: 350px;
 @import './src/styles/ceph-custom/forms';
 @import './src/styles/ceph-custom/grid';
 @import './src/styles/ceph-custom/icons';
+@import './src/styles/ceph-custom/navs';
+@import './src/styles/ceph-custom/toast';
 
 /* If javascript is disabled. */
 .noscript {
   padding-top: 5em;
 }
 
-.noscript p {
-  color: $color-noscript-text;
-}
-
-/* Card */
-.card-footer button.btn:not(:first-child) {
-  margin-left: 5px;
-}
-/* Modal dialog */
-.modal-footer button.btn:not(:first-child) {
-  margin-left: 5px;
-}
-
-.nav-tabs {
-  margin-bottom: 15px;
-}
-
-/* Block UI */
-.block-ui-wrapper {
-  background: $color-transparent-black !important;
-}
-
-.tooltip-inner {
-  background-color: $color-solid-white;
-  border: 1px solid $color-gray20;
-  color: $color-gray13;
-  font-size: 1.1em;
-}
-
-.toast-message > ul {
-  margin: 0;
-  padding-left: 1em;
-}
-
-.cd-header,
-legend {
-  @extend .pb-1;
-  @extend .mt-4;
-  @extend .mb-4;
-  @extend .border-bottom;
-}
-
+// TODO: Replace (cd-)col-form-label with something more generic.
```

Comment:
@SchoolGuy @tspmelo I understand that this is to be addressed in a follow-up PR, right?

Reply:
Please make sure to create tracker issues for these open tasks and refer to them in the comment.

Reply:
Tracker ticket created: https://tracker.ceph.com/issues/46870